packages/core/src/tracing/backend_mappers/index.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+module.exports = {
+  get transform() {
```

Comment:
https://github.com/instana/nodejs/pull/1448#discussion_r1865736225
I cannot see the refactoring from ^ 🤔 
Its still the same code than before. Please use ^

Reply:
Discussed: We keep it as it is. Its preference